plugins/org.jkiss.dbeaver.ext.clickhouse/src/org/jkiss/dbeaver/ext/clickhouse/model/ClickhouseTable.java

```diff
@@ -147,6 +150,12 @@ public String generateTableUpdateBegin(String tableName) {
         return "ALTER TABLE " + tableName + " UPDATE ";
     }
 
+    @Override
+    public DBSObject refreshObject(@NotNull DBRProgressMonitor monitor) throws DBException {
+        ((ClickhouseSchema) getSchema()).resetStatistics();
```

Comment:
Please see the previous note first. By doing so, we can move this logic to the very base class.